File-Formats/PDF/Convert-HTML-To-PDF/azure.md

```diff
@@ -7,60 +7,62 @@ documentation: UG
 keywords: Assemblies
 ---
 
-# HTML to PDF conversion in Azure
+# Convert HTML to PDF file in Azure
 
 The Syncfusion [HTML to PDF converter](https://www.syncfusion.com/document-processing/pdf-framework/net/html-to-pdf) is a .NET library for converting webpages, SVG, MHTML, and HTML to PDF using C#. It is reliable and accurate. The result preserves all graphics, images, text, fonts, and the layout of the original HTML document or webpage. Using this library, you can convert an HTML to PDF using C# with the Blink rendering engine in Azure App Service on Linux, Azure app service using the Linux [docker](https://www.docker.com/why-docker) container and Azure Function Application Linux.
 
 ## Azure App Service Linux
 
 ### Steps to convert HTML to PDF in Azure App service on Linux:
 
-Create a new ASP.NET Core MVC application. 
+1. Create a new ASP.NET Core MVC application.
 ![Convert HTMLToPDF Azure NetCore Step1](htmlconversion_images/AzureNetCore1.png)
+
+2. Choose your project's target framework, then select     Configure for HTTPS.
```

Comment:
1. Remove the unwanted space. 
2. Modify the sentence like "Choose your project's target framework and select Configure for HTTPS."

Reply:
Modified the content